### DISCUSSION TITLE
Run Current Cell Command in VScode

### DISCUSSION THREAD

**Discussion by synthds** [quarto-dev/quarto-cli#5911] [Q&A]
### Description

When sending code to run using ctrl+shift+enter (run entire cell) the cell won't execute if there is more than one line of code.

E.g.,

This will run:
```{python}
print("This will run")
```

This WON'T run (in fact the run cell button is completely unresponsive

```{python}
print("this")
print("wont run")
```

Any ideas? This was not an issue I was having just a few weeks ago.

**Comment by mcanouil** [comment]
We need a bit more information than a keyboard shortcut here in order to help 😅

Which IDE? Which version? Which operating system? Jupyter Notebook or flat Qmd file? R code only?

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by cscheid** [reply]
> Which IDE?

That's on the discussion title (it's VS Code).

But, in any case, bug reports for VS Code issues belong in https://github.com/quarto-dev/quarto/.

**Reply by mcanouil** [reply]
> That's on the discussion title (it's VS Code).

Indeed, I too, can have slow day 😅
Although, I was not able to reproduce on a simple example. 🤷‍♂️
